Atomic/atomic.py

```diff
@@ -645,36 +645,62 @@ def gen_cmd(self, cargs):
             args.append(c)
         return " ".join(args)
 
+    def get_fq_name(self, image_info):
+        if len(image_info['RepoTags']) > 1:
+            raise ValueError("\n{} is tagged with multiple repositories. "
+                             "Please use a repository name instead.\n".format(self.image))
+        else:
```

Comment:
what I don't understand is what the tag means in *fully qualify* an image; by fully qualify an image I'm referring to prepend to the image name the registry part